### PR TITLE
Set version to 1.0.0

### PR DESCRIPTION
**Pull request by cole-h**

### CODE REVIEW

**Review by grahamc — APPROVED** [review]

File: flake.nix [hunk 1]
```diff
@@ -95,7 +95,7 @@
 
             sharedAttrs = {
               pname = "riff";
-              version = "0.0.0-unreleased";
+              version = "1.0.0";
               src = self;
```
Comment (grahamc) [inline flake.nix:99]: It occurs to me these details could be loaded with the builtin toml reader?

Comment (grahamc) [reply flake.nix:99]: this is a random thought, not a thing.

**Comment by grahamc** [comment]
Before merging I want to fetch the artifacts and try them out a bit.

File: flake.nix (hunk 1 above)
Comment (cole-h) [reply flake.nix]: It's a good idea. Done.

**Comment by cole-h** [comment]
(Disabled automerge so we can get https://github.com/DeterminateSystems/riff/pull/110 in first.)

**Pull request merged**